src/eduid_idp/config.py

```diff
@@ -78,7 +79,7 @@
                     'default_scoped_affiliation': None,
                     'vccs_url': 'http://localhost:8550/',  # VCCS backend URL
                     'insecure_cookies': '0',  # Set to 1 to not set HTTP Cookie 'secure' flag
-                    'actions_auth_shared_secret': 'abcdef',
+                    'actions_auth_shared_secret': 's9Wwg13haLFk4y4joWU7MFdAmfe7SdjY',  # exactly 32 bytes long
```

Comment:
That looks way too much like a real password to be committed to the repository =). I don't think a default shared secret makes any sense. Better to fail actions if none is configured.